bundles/ch.elexis.core.findings.util/src/ch/elexis/core/findings/util/fhir/transformer/AppointmentTerminTransformer.java

```diff
@@ -68,10 +73,16 @@ public Optional<Appointment> getFhirObject(IAppointment localObject, SummaryEnum
 
 	@Override
 	public Optional<IAppointment> updateLocalObject(Appointment fhirObject, IAppointment localObject) {
-
+		historyService = AppointmentHistoryServiceHolder.get();
+		String originalReason = localObject.getReason();
+		LocalDateTime originalEndTime = localObject.getEndTime();
 		attributeMapper.fhirToElexis(fhirObject, localObject);
-		// TODO more
-
+		if (!originalEndTime.equals(localObject.getEndTime())) {
+			historyService.logAppointmentDurationChange(localObject, originalEndTime, localObject.getEndTime());
+		}
+		if (!originalReason.equals(fhirObject.getDescription())) {
+			historyService.logAppointmentEdit(localObject);
+		}
```

Comment:
Fehlt hier nicht der Fall das sich Start- und End- Zeit ändert. Also ein Verschieben des Termins.

Reply:
Nein den verschieben im gleichen Bereich ist mit dem gewährleistet

```
		if (!originalEndTime.equals(localObject.getEndTime())) {
			historyService.logAppointmentDurationChange(localObject, originalEndTime, localObject.getEndTime());
		}
```

und das Verschieben in einem neun Bereich, das ist SlotTerminTransformer.java erfasst